Sources/BraintreeLocalPayment/BTLocalPaymentRequest.swift

```diff
@@ -9,7 +9,12 @@ import BraintreeDataCollector
 #endif
 
 /// Used to initialize a local payment flow
-@objcMembers public class BTLocalPaymentRequest: NSObject {
+/// The POST body for v1/local_payments/create
+@objcMembers public class BTLocalPaymentRequest: NSObject, Encodable {
```

Comment:
Take it or leave it but we could also consider separating the encodable portion out into a new file, we've done that in most places where we've made this change, like here for example: https://github.com/braintree/braintree_ios/blob/main/Sources/BraintreeSEPADirectDebit/SEPADebitAccountsRequest.swift

Reply:
Sure, updated: [3e00b0e](https://github.com/braintree/braintree_ios/pull/1468/commits/3e00b0e96d9cd3d3ca2e5601a1bc8cf79acb3d31) what do you think??